docs/user-documentation/pss.md

```diff
@@ -1,10 +1,181 @@
 ---
 title: Postal Service over Swarm
-hide_title: true
 id: pss
 slug: /pss
 sidebar_label: Postal Service over Swarm
 ---
 
 import Tabs from '@theme/Tabs'
 import TabItem from '@theme/TabItem'
+
+Swarm provides the ability to send messages that appear to be normal Swarm traffic, but are in fact messages that may be received and decrypted to reveal their content only by the specific nodes they were intended to be received by.
```

Comment:
Suggestion: `... only to specific nodes that were intended to receive them`

Reply:
```suggestion
Swarm provides the ability to send messages that appear to be normal Swarm traffic, but are in fact messages that may be received and decrypted to reveal their content only to specific nodes that were intended to receive them.
```